content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -64,34 +62,26 @@ Choose this option if your cluster doesn't have [Tiller](https://github.com/kube
 1. Make an Istio working directory for fetching the charts:
 
     {{< text bash >}}
-
     $ mkdir -p $HOME/istio-fetch
-
     {{< /text >}}
 
 1. Fetch the helm templates needed for installation:
 
     {{< text bash >}}
-
     $ helm fetch istio.io/istio-init --untar --untardir $HOME/istio-fetch
     $ helm fetch istio.io/istio --untar --untardir $HOME/istio-fetch
-
     {{< /text >}}
 
 1. Create a namespace for the `istio-system` components:
 
     {{< text bash >}}
-
     $ kubectl create namespace istio-system
-
     {{< /text >}}
 
 1. Install all the Istio's [Custom Resource Definitions or CRDs for short](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) via `kubectl apply`, and wait a few seconds for the CRDs to be committed to the Kubernetes API server:
 
     {{< text bash >}}
-
     $ helm template $HOME/istio-fetch/istio-init --name istio-init --namespace istio-system | kubectl apply -f -
```

Comment:
This needs the flag to for demo installs I think

Reply:
I did not do anything special here and it worked.

Reply:
You are right I misread this. Thanks!